bionic/aip/task.py

```diff
@@ -0,0 +1,156 @@
+""" Data model for task running on AI platform
+"""
+import logging
+import os
+import sys
+from dataclasses import dataclass
+from typing import Callable, Optional
+
+import blocks
+import cloudpickle
+import googleapiclient.discovery
+from bionic.aip.future import Future
+from bionic.aip.tune import Tune
+
+
+@dataclass(frozen=True)
```

Comment:
TIL `dataclass`! This looks pretty good but it seems to be added in Python 3.7. Bionic supports Python 3.6 as well, so you'll have to replace this with [attrs](https://www.attrs.org/en/stable/) classes. We already include `attrs` as a required dep in Bionic, so you don't have to worry about installing it.

Reply:
Extending this comment, does `sq-blocks` also support python 3.6+?

Reply:
Yes blocks is 2+ including 3.6, i'll remove the dataclass use here